.changeset/late-rings-poke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/textarea': minor
```

Comment:
It's a good question. If this was a peer dependency, maybe, because the dependency management kind of falls on the consumer and you need a way to tell them to do something.

As it's a dependency and not re-exported, it's bundled into the compiled code and distributed, so you shouldn't have any idea about the change. You can't directly get at it to use it, it's an internal implementation detail.

If we were treating it as a library and re-exporting it, then yup, that'd be a major.

Reply:
wait. so what should it be?

Reply:
@shleewhite Is `@twilio-paste/textarea-resize-library` a peer dependency of `@twilio-paste/textarea`? 
If yes I think the answer to the question is yes, it should be major.

@SiTaggart lmk if that isn't correct ☝️

Reply:
Correct, you are adding a peer dependency to the package, which won't be installed for the consumer automatically, and as such you should mark it as a major

Reply:
(I realise my wording was all jumbled up in my original explanation. I tried not to give the answer and made it more confusing)